frontend/src/pages/dashboard/Dashboard.scss

```diff
@@ -217,7 +223,7 @@
 			align-items: center;
 			display: flex;
 			gap: var(--spacing);
-			height: calc(var(--header-height) - var(--spacing));
+			// height: calc(var(--header-height) - var(--spacing));
```

Comment:
The Code is not used, is it still needed or relevant or can it be safely removed or deleted?

Reply:
it may be needed later on